docs/jitpack_dependencies.md

```diff
@@ -3,22 +3,52 @@
 If you like to live on the bleeding edge, [jitpack.io](https://jitpack.io) can be used to obtain SNAPSHOT versions.
 Use the following dependency description instead:
 
-```xml
-<dependency>
-    <groupId>com.github.testcontainers.testcontainers-java</groupId>
-    <artifactId>--artifact name--</artifactId>
-    <version>-SNAPSHOT</version>
-</dependency>
-```
-
-A specific git revision (such as `093a3a4628`) can be used as a fixed version instead. The JitPack maven repository must also be declared, e.g.:
-
-```xml
-<repositories>
-    <repository>
-        <id>jitpack.io</id>
-        <url>https://jitpack.io</url>
-    </repository>
-</repositories>
-```
+=== "Gradle"
```

Comment:
let's make Maven go first

Reply:
In other parts of the docs we have Gradle first as well (at least the instances I found), so this would be more consistent.

Reply:
Ha! You are right, I didn't realize that. Sorry for the confusion, @YannickWeber!

Reply:
swapped it again 🤷